config.go

```diff
@@ -22,41 +22,44 @@ import (
 	"github.com/decred/dcrd/dcrutil"
 	"github.com/decred/dcrd/hdkeychain"
 	"github.com/decred/dcrstakepool/internal/version"
-	flags "github.com/jessevdk/go-flags"
+	"github.com/jessevdk/go-flags"
 )
 
 const (
-	defaultBaseURL         = "http://127.0.0.1:8000"
-	defaultClosePoolMsg    = "The voting service is temporarily closed to new signups."
-	defaultConfigFilename  = "dcrstakepool.conf"
-	defaultDataDirname     = "data"
-	defaultLogLevel        = "info"
-	defaultLogDirname      = "logs"
-	defaultLogFilename     = "dcrstakepool.log"
-	defaultCookieSecure    = false
-	defaultDBHost          = "localhost"
-	defaultDBName          = "stakepool"
-	defaultDBPort          = "3306"
-	defaultDBUser          = "stakepool"
-	defaultListen          = ":8000"
-	defaultPoolEmail       = "admin@example.com"
-	defaultPoolFees        = 7.5
-	defaultPoolLink        = "https://forum.decred.org/threads/rfp-6-setup-and-operate-10-stake-pools.1361/"
-	defaultPublicPath      = "public"
-	defaultTemplatePath    = "views"
-	defaultSMTPHost        = ""
-	defaultMaxVotedTickets = 1000
-	defaultDescription     = ""
-	defaultDesignation     = ""
+	defaultBaseURL                  = "http://127.0.0.1:8000"
+	defaultClosePoolMsg             = "The voting service is temporarily closed to new signups."
+	defaultConfigFilename           = "dcrstakepool.conf"
+	defaultStakepooldConfigFilename = "stakepoold.conf"
```

Comment:
Don't think this is needed anymore. Remove this and related additions to this file. Remember to `go fmt ./...` afterwards.

Reply:
changes applied.